### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Select Subscriptions_ Keyboard: Keyboard focus is not visible on list items in “High Contrast” mode.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3338]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.1.1; #Keyboard; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** 2.4.7
**NOTE: Same issue is observed in “Dracula” mode.**

**Repro Steps:**
1) Open IntelliJ IDEA application
2) Press 'Ctrl +Alt+ S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Select Device login and tab till Sign in and press enter
8) Tab till 'Copy and Open' button and press enter
9) Paste the code in the browser and click on next.
10)Open the application again and navigate to all list items.
11)Verify keyboard focus is visible on list items or not.

**Actual Result:**
Keyboard focus is not visible on list items in “High Contrast” mode.

**Expected Result:**
Keyboard focus should be  visible on list items in “High Contrast” mode.

**User Impact:**
Keyboard users find difficulty in understanding where the focus is if focus is not visible on controls.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}
![Bug_3338_Keyboard issue](https://user-images.githubusercontent.com/38523250/62205584-64996780-b3ad-11e9-9233-45974dfff1a7.png)

**Comment by Flanker32** [comment]
@Accessibility56 The behavior of focused item is controlled by IntelliJ, which is out of the scope of our toolkit. An here is a snapshot of IntelliJ Path variables, which behaves same as us.

![3338](https://user-images.githubusercontent.com/12445236/64506539-80781d80-d30a-11e9-92fe-ceff40291f11.gif)

As we can see, keyboard focus is not visible on table, but users could use 'Up' or 'Down' to select items when focus is on table.

**Comment by Flanker32** [comment]
- Importance of experience: Medium, users will not know whether table is on focus by tab
- Quality of experience: Little, use could reach the next component and pree `Shift`+`Tab` to get back to table, and then they could access each item by `Up` or `Down`
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue of IntelliJ which has been linked to AAT, please help us verify it , thanks!

**Comment by Accessibility56** [comment]
Logged an external bug with IntelliJ https://youtrack.jetbrains.com/issue/IDEA-222329

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749786
